src/logic/hooks/__tests__/useCanTxExecute.test.ts

```diff
@@ -0,0 +1,127 @@
+import { calculateCanTxExecute } from '../useCanTxExecute'
+
+describe('useCanTxExecute tests', () => {
```

Comment:
Are you not going to test the hook?

Reply:
I was thinking of leaving it like this as all the logic is in the `calculateCanTxExecute`.

Reply:
Perhaps rename this then:

```ts
describe('useCanTxExecute tests', () => {
```

Reply:
I left the outer `describe` for the hook and set a nested for the function
https://github.com/gnosis/safe-react/blob/144a35c22563570b0f4470ddb854ec859d6238cd/src/logic/hooks/__tests__/useCanTxExecute.test.ts#L3-L4

Should I remove the outer one?

Reply:
Ah, I didn't see that on the small preview - no worries.

I still personally think the hook should be tested as it was added it in the PR. You could reuse a lot of the test logic.